demo-app/src/main/kotlin/io/getstream/video/android/App.kt

```diff
@@ -54,6 +54,6 @@ class App : Application() {
     }
 }
 
-const val API_KEY = BuildConfig.API_KEY
+val STREAM_SDK_ENVIRONMENT = if (BuildConfig.FLAVOR == "production") "pronto" else "demo"
```

Comment:
We have a new constant for flavors, you can use `StreamFlavors.production` instead of the string.